src/vos/vos_obj_cache.c

```diff
@@ -160,18 +160,19 @@ vos_obj_cache_create(int32_t cache_size, struct daos_lru_cache **occ)
 {
 	int	rc;
 
-	D_DEBUG(DB_TRACE, "Creating an object cache %d\n", (1 << cache_size));
 	rc = daos_lru_cache_create(cache_size, D_HASH_FT_NOLOCK,
 				   &obj_lru_ops, occ);
-	if (rc)
-		D_ERROR("Error in creating lru cache: "DF_RC"\n", DP_RC(rc));
+	DL_CDEBUG(rc != 0, DLOG_ERR, DLOG_INFO, rc, "Create VOS object LRU cache %p, size %u",
+		  *occ, 1 << cache_size);
+
```

Comment:
I'm not sure if it's a good idea to introduce such D_INFO logs. Discussed the ticket with @Nasf-Fan offline, seems some CR code access/modify sys_db without locking, which result in race on accessing sysdb (sysdb is accessed by all xstreams, locking is required).

Let's hold on landing this PR for this moment.

Reply:
We can know the LRU cache address and size via such log, that will help us to trace the LRU cache life-cycle.

On the other hand, I have already refreshed the patch to hold the lock against vos sys_db before accessing it.

Reply:
Other changes looks good to me. Could you change the DLOG_INFO to DB_TRACE if you need to collect the debug information?

Reply:
Anyway, we can check the issue is fixed after applying the lock changes or not. If failed, then add more debug log.
Refreshed the patch.